core/tauri/src/window/scripts/drag.js

```diff
@@ -63,4 +63,9 @@
       }
     })
   }
+  if ('adoptedStyleSheets' in document) {
```

Comment:
```suggestion
  if (osName === 'windows' && 'adoptedStyleSheets' in document) {
```

Reply:
We might get `app-region: drag` in the future on Linux and Mac, do we want to limit this part to only run on Windows?

Reply:
yes, until other platforms support, we should limit to only Windows